plugins/platform-ui/src/routes.ts

```diff
@@ -313,18 +309,31 @@ export class Router<T> implements ApplicationRouter<T> {
     return this.calcFragment(vars, this.parents(), this.children())
   }
 
-  private calcFragment (vars: Partial<T>, parents: Router<any>[], children: Router<any>[]): string {
+  /**
+   * Will return last one defined fragment.
+   */
+  private calcFragment (vars: Partial<T>, parents: Array<Router<any>>, children: Array<Router<any>>): string {
     let variables = this.variables as T
-    if (vars) {
-      variables = { ...variables, ...vars }
+    variables = { ...variables, ...vars }
+    // Check all children in reverse order
+    for (const r of children.reverse()) {
```

Comment:
I'm not sure about details, but just want to notice that `reverse` is mutable and that's probably not what we want from `calcFragment`.

Reply:
Thanks, reverse mutation was unexpected by me :-(